AITS/TemperatureAIT.C

```diff
@@ -0,0 +1,82 @@
+#include "TemperatureAIT.H"
+
+using namespace Foam;
+
+preciceAdapter::AITS::Temperature::Temperature
+(
+    const Foam::fvMesh& mesh,
+    const std::string nameT
+    )
+:
+T_(
+    const_cast<volScalarField*>
+    (
+        &mesh.lookupObject<volScalarField>(nameT)
+    )
+    ),
+    mesh_(mesh)
+{
+    dataType_ = scalar;
+}
+
+void preciceAdapter::AITS::Temperature::write(double * buffer, bool meshConnectivity)      //Bernhard Version 2
```

Comment:
I am not sure why this comment is here, but it does not look necessary.
```suggestion
void preciceAdapter::AITS::Temperature::write(double * buffer, bool meshConnectivity)
```